### DISCUSSION TITLE
api 能否使用 POST 请求提交列表嵌套数据

### DISCUSSION THREAD

**Discussion by zhibeen** [apiflask/apiflask#157] [Q&A] [answered]
POST 方法 能否上传 List  ，list 中的每组要通过Schemas 检验 

类似于 output 中的 List(Nested(PetSchema))

**Comment by greyli** [comment, marked as answer]
可以，你只需要在 input 模式里将对应的字段定义为 List(Nested(PetSchema)) 就可以了。
